CleverPush/CleverPush/Source/CleverPush.h

```diff
@@ -125,15 +125,20 @@ extern NSString * const CLEVERPUSH_SDK_VERSION;
 
 + (NSArray*)getAvailableTags __attribute__((deprecated));
 + (NSArray*)getAvailableTopics __attribute__((deprecated));
-+ (NSArray*)getSubscriptionTags;
++ (NSArray<NSString*>*)getSubscriptionTags;
 + (NSArray<CPNotification*>*)getNotifications;
 + (void)getNotifications:(BOOL)combineWithApi callback:(void(^)(NSArray<CPNotification*>*))callback;
 + (void)getNotifications:(BOOL)combineWithApi limit:(int)limit skip:(int)skip callback:(void(^)(NSArray<CPNotification*>*))callback;
 + (void)removeNotification:(NSString*)notificationId;
+
++ (NSArray<NSString*>*)getSeenStories;
++ (NSMutableArray<NSString*>*)getSubscriptionTopics;
```

Comment:
Please remove this line and modify below line:

https://github.com/cleverpush/cleverpush-ios-sdk/pull/79/files#diff-c57061cf618b77e65c18a4ff65ab9d99f144c135be7021da3333fe304eb28bf0R139